runtime/mangata-kusama/src/lib.rs

```diff
@@ -190,7 +190,7 @@ mod currency {
 	pub const DOLLARS: Balance = super::UNIT;
 
 	pub const fn deposit(items: u32, bytes: u32) -> Balance {
-		items as Balance * 15 * CENTS + (bytes as Balance) * 6 * CENTS
+		items as Balance * 5000 * DOLLARS + (bytes as Balance) * 60 * CENTS
```

Comment:
roughly 1000x multiple of
```	pub fn base_tx_in_mgr() -> Balance {
		UNIT
	}
``` 

for reference Polkadot:
```items as Balance * 20 * DOLLARS + (bytes as Balance) * 100 * MILLICENTS```
&
base_tx_in_ksm == CENT / 10